From 7c3aa26a536ec2f3454a42c33cb1ce24a20cd776 Mon Sep 17 00:00:00 2001
From: Emmanuel Jillela <emmanuel.jillela@intel.com>
Date: Tue, 21 Jul 2015 10:40:37 -0700
Subject: [PATCH 141/441] Corrected sram size of mbox channel from npcpu to
 appcpu. and fixed a compiltaion error seen when hw mbox debug log enabled.

---
 arch/x86/hw_mailbox/hw_mbox_drv.c | 2 +-
 include/linux/hw_mbox_defs.h      | 4 ++--
 2 files changed, 3 insertions(+), 3 deletions(-)

--- a/arch/x86/hw_mailbox/hw_mbox_drv.c
+++ b/arch/x86/hw_mailbox/hw_mbox_drv.c
@@ -4175,7 +4175,7 @@ static int __init hwMbox_drv_init(void)
         HW_MBOX_DISABLE_ALL_INTERRUPTS();
 
         DPRINTK("APP-CPU: intrMaskRegs(H L): 0x%08x 0x%08x",
-                HW_MBOX_MASTER_INTR_MASK_H_REG_GET()
+                HW_MBOX_MASTER_INTR_MASK_H_REG_GET(),
                 HW_MBOX_MASTER_INTR_MASK_L_REG_GET());
 
         DPRINTK("APP-CPU: Enabling all interrupts!");
--- a/include/linux/hw_mbox_defs.h
+++ b/include/linux/hw_mbox_defs.h
@@ -763,9 +763,9 @@ typedef Uint8 __hwMbox_opcodeMessageSize
     ENTRY(    4  , HW_MBOX_MASTER_PHY_CPU              , HW_MBOX_MASTER_NP_CPU               , hw_mbox_event            , 8         )    \
     ENTRY(    5  , HW_MBOX_MASTER_MOCA_C               , HW_MBOX_MASTER_NP_CPU               , hw_mbox_event            , 8         )    \
     ENTRY(    6  , HW_MBOX_MASTER_NP_CPU               , HW_MBOX_MASTER_BBU_C                , hw_mbox_opcode           , 8         )    \
-    ENTRY(    7  , HW_MBOX_MASTER_NP_CPU               , HW_MBOX_MASTER_APP_CPU              , hw_mbox_opcode           , 8         )    \
+    ENTRY(    7  , HW_MBOX_MASTER_NP_CPU               , HW_MBOX_MASTER_APP_CPU              , hw_mbox_opcode           , 64        )    \
     ENTRY(    8  , HW_MBOX_MASTER_NP_CPU               , HW_MBOX_MASTER_PHY_CPU              , hw_mbox_opcode           , 8         )    \
-    ENTRY(    9  , HW_MBOX_MASTER_NP_CPU               , HW_MBOX_MASTER_MOCA_C               , hw_mbox_opcode           , 64        )    \
+    ENTRY(    9  , HW_MBOX_MASTER_NP_CPU               , HW_MBOX_MASTER_MOCA_C               , hw_mbox_opcode           , 8         )    \
     ENTRY(   10  , HW_MBOX_MASTER_APP_CPU              , HW_MBOX_MASTER_NP_CPU               , hw_mbox_opcode           , 64        )
 
 /* static channel Table Expanders: */
